### PR TITLE
Minutes

### PR DESCRIPTION
**Pull request by japaric**

### CODE REVIEW

File: minutes/2019-12-03.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+* japaric
+* therealprof
+* rubberduck203
+* disasm
+* mabez
+* cr1901
+* ryankurte
+* nomaxx117 (GH: Noah-Kennedy)
+* jamesmunns (on and off)
+* thejpster (late)
```
Comment (jonas-schievink) [inline minutes/2019-12-03.md]: :wave:

File: minutes/2019-12-03.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+## Last Week's Agenda
+
+* Mutex Trait Naming
+    * Consensus on naming: `mutex-trait`
+* https://github.com/rust-embedded/embedded-hal/issues/156
+    * Suggestion to base this on musca-b1 because it's shiny and new and Cortex-M33
+* https://github.com/rust-embedded/embedded-hal/pull/171
+    * Discussion about whether it's a good idea to use the heavily async-await based contract `core::task::Poll` instead of `nb`
+        * Pro: Use of core components instead of custom crates
+        * Con: Maybe not idea for non-async use cases
```
Comment (jonas-schievink) [inline minutes/2019-12-03.md]:
```suggestion
        * Con: Maybe not ideal for non-async use cases
```

File: minutes/2019-12-03.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+* Meetings: Tuesday 8pm Europe/Berlin time
+* [Join the Chat]
+* Today's meeting: 2019-12-03
+* [Meeting Agenda]
+* [Next Week's Meeting Agenda]
+
+[Coordination Repository]: https://github.com/rust-embedded/wg
+[Join the Chat]: https://riot.im/app/#/room/#rust-embedded:matrix.org
+[Meeting Agenda]: https://github.com/rust-embedded/wg/issues/398
+[Next Week's Meeting Agenda]: #
```
Comment (jamesmunns) [inline minutes/2019-12-03.md]:
```suggestion
[Next Week's Meeting Agenda]: https://github.com/rust-embedded/wg/issues/403
```

File: minutes/2019-12-03.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+* https://github.com/rust-embedded/cortex-m-rt/issues/212
+    * Requires a new release to fix soundness issue
+        * Consensus on releasing as a ~minor~ patch release (v0.6.x) potentially causing breakage upon abuse of the unsoundness
+        * No consensus on yanking older versions of 0.6
+        * As a bonus: New release will yield reproducible builds
+
+## Last Week's Action Items
+
+* [ ] @jamesmunns to write a proposal about a monorepo for books (still todo)
+* [ ] Someone to add "update your cortex-m" with instructions to the blog posts
```
Comment (jonas-schievink) [inline minutes/2019-12-03.md]:
```suggestion
* [ ] Someone to add "update your cortex-m-rt" with instructions to the blog posts
```

File: minutes/2019-12-03.md (hunk 1 above)
Comment (jamesmunns) [reply minutes/2019-12-03.md]: You gotta add yourself on the hackmd next time!

File: minutes/2019-12-03.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+* japaric
+* therealprof
+* rubberduck203
+* disasm
+* mabez
+* cr1901
+* ryankurte
+* nomaxx117 (GH: Noah-Kennedy)
+* jamesmunns (on and off)
+* thejpster (late)
```
Comment (jamesmunns) [inline minutes/2019-12-03.md]:
```suggestion
* thejpster (late)
* jonas-schievink
```

Comment (jonas-schievink) [reply minutes/2019-12-03.md]: Oh, multi-line suggestions work now?

**Comment by japaric** [comment]
bors r=jamesmunns

**Pull request merged**